### PR TITLE
refactoring and cleanup

### PR DESCRIPTION
**Pull request by laicar**

### CODE REVIEW

File: app/src/main/java/c4stor/com/feheroes/activities/collection/CollectionActivity.java [hunk 1]
```diff
@@ -408,11 +413,11 @@ public View getView(final int position, View convertView, ViewGroup parent) {
             holder.collName.setText(hero.getDisplayName(getContext()));
 
 
-            StringBuilder sb = new StringBuilder();
+            /*StringBuilder sb = new StringBuilder();
             for (int i = 1; i <= hero.stars; i++) {
```
Comment (nicolasdalsass) [inline app/src/main/java/c4stor/com/feheroes/activities/collection/CollectionActivity.java]: Don't comment useless code, delete it ! 🚀

File: app/src/main/java/c4stor/com/feheroes/activities/ivcheck/IVCheckActivity.java [hunk 2]
```diff
@@ -145,10 +152,10 @@ public void onNothingSelected(AdapterView<?> parent) {
             }
         });
 
-        populateSpinner(R.id.spinner_stars, R.array.stars_array);
+        populateSpinner();
 
         adAdBanner();
-//        disableAdBanner();
+        disableAdBanner();
```
Comment (nicolasdalsass) [inline app/src/main/java/c4stor/com/feheroes/activities/ivcheck/IVCheckActivity.java]: Please don't commit both adAdBanner and disableAdBanner uncommented, I can get banned from the play store for displaying invisible ads ^^

**Review by nicolasdalsass — CHANGES_REQUESTED** [review]
LGTM ! A small modification required, as a matter of good practice :)

File: app/src/main/java/c4stor/com/feheroes/activities/ivcheck/IVCheckActivity.java (hunk 2 above)
Comment (laicar) [reply app/src/main/java/c4stor/com/feheroes/activities/ivcheck/IVCheckActivity.java]: My bad, I didn't catch it

**Pull request merged**